code/lib/core-server/src/presets/common-preset.ts

```diff
@@ -232,3 +240,60 @@ export const managerHead = async (_: any, options: Options) => {
 
   return '';
 };
+
+const WHATS_NEW_CACHE = 'whats-new-cache';
+const WHATS_NEW_URL = 'https://storybook-dx.netlify.app/.netlify/functions/whats-new';
+
+// Grabbed from the implementation: https://github.com/storybookjs/dx-functions/blob/main/netlify/functions/whats-new.ts
+type WhatsNewResponse =
+  | { title: string; url: string; publishedAt: string; excerpt: string }
+  | 'Post not found';
```

Comment:
@winkerVSbecks This error response is a bit off. Probably better to return a standard 4xx or 5xx, so that I only enable the future when I get back a 2xx response

Reply:
sure, I can do that.